app/views/articles/tags/_meta.html.erb

```diff
@@ -3,15 +3,15 @@
      timeframe: params[:timeframe],
      content_for: true,
    ) %>
-<link rel="canonical" href="https://dev.to/t/<%= @tag %>" />
+<link rel="canonical" href="<%= ApplicationConfig["APP_PROTOCOL"] %><%= ApplicationConfig["APP_DOMAIN"] %>/t/<%= @tag %>" />
```

Comment:
For this and the others I think you can use the new helper in 

https://github.com/thepracticaldev/dev.to/blob/209c01037a18d5ac1e0ed8995029e93f60773cfe/app/helpers/application_helper.rb#L168

Basically it would become:

```erb
<%= app_url("/t/#{@tag}") %>
```

Reply:
missed this, thanks will use it!

Reply:
@rhymes This is probably common enough that we could add it to `app/lib/url.rb` as 

```ruby
def self.tag(tag)
  url("/t/#{tag}"
end
```

LMK if I should make the PR for it.

Reply:
Good call @citizen428, maybe it can be introduced with this PR if @Ridhwana wants to add this too?

Reply:
I'll add this too!

Reply:
My PR #6771 introduces `self.tag`, so maybe try and get that in first?

Reply:
Ok, will do 👍